assets/constants.ts

```diff
@@ -5679,6 +5679,9 @@ export default {
   // A regular expression for allowed characters in URL fragment fields.
   "VALID_URL_FRAGMENT_REGEX": "^[a-z]+(-[a-z]+)*$",
 
+  // A regular expression for allowed characters for thumbnail filename.
+  "VALID_THUMBNAIL_FILENAME": "^[^.](?!.*/)(?!.*\\.\\.).*.svg$",
```

Comment:
Can we rename this to `VALID_THUMBNAIL_FILENAME_REGEX`? Also seems like this RegEx allows consecutive periods, but we do not want to allow that.